flask_restful/inputs.py

```diff
@@ -242,12 +242,15 @@ def boolean(value):
     if type(value) == bool:
         return value
 
+    if value == '':
+        return None
+
     if not value:
         raise ValueError("boolean type must be non-null")
     value = value.lower()
-    if value in ('true', '1',):
+    if value in ('true', '1', 1):
```

Comment:
How did you determine that this is necessary? Can you add a test for this? I ask because you can never get to this point in the code if `value` is an `int` since an `AttributeError` will be thrown on the line above since `int` has no method `lower()`.